### PR TITLE
Introduce a Timesketch roadmap doc and Sigma

### LINKED ISSUE #1902: Create a Sigma roadmap / vision document
Based on some recent features, there where questions ideas where the Sigma integration in Timesketch might lead to. We should write down what the current idea for the future is and how it all ties in to the Timesketch workflow.

### PR DESCRIPTION
**Pull request by jaegeral**
Introduce a Roadmap document and first start with a Sigma section on what our plan is moving forward

closes #1902

### CODE REVIEW

File: docs/developers/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+This document will give a human readable view on the planned future for Timesketch.
+The actual implementation of the dedicated feature is tracked and discussed in individual Github Issues.
+ 
+# Fineprint
+ 
+As much as we try to stick to the roadmap, the development of the project will remain fluid and drastic changes in resource allocation might happen, new ideas might pop up or old ideas might become obsolete.
+ 
+# How to contribute to the roadmap
+ 
+We invite users, developers from the entire community to provide feedback via e.g. [Twitter](https://twitter.com/TimesketchProj), Slack and Github issues. To directly contribute to this roadmap create a PR and propose changes and explain the rationale behind your proposed changes.
```
Comment (berggren) [inline docs/developers/roadmap.md:12]:
```suggestion
We invite users, developers from the entire community to provide feedback via e.g [Github issues] (https://github.com/google/timesketch/issues/new/choose), [Slack] (https://timesketch.org/community/resources/) or [Twitter](https://twitter.com/TimesketchProj)

To directly contribute to this roadmap create a PR and propose changes and explain the rationale behind your proposed changes.
```

**Review by berggren — CHANGES_REQUESTED** [review]
One small nit, otherwise LG

File: docs/developers/roadmap.md (hunk 1 above)
Comment (jaegeral) [reply docs/developers/roadmap.md]: Ack

**Comment by jaegeral** [comment]
Ok seems I can not merge without your approval, even if I dismiss your review…. @berggren

**Pull request merged**